backend/src/test/kotlin/pl/akai/fillist/web/spotifywrapper/SpotifyPlaylistsTests.kt

```diff
@@ -52,4 +61,39 @@ class SpotifyPlaylistsTests {
         // assertEquals(playlist.description, createPlaylistRequestBody.description)
         assertEquals(playlist.owner.id, userId)
     }
+
+    @Test
+    fun getPlaylistDetails() {
+        // given
```

Comment:
Comments quickly become outdated, making them less helpful for understanding code. They can also make code more difficult to understand, especially if they are not updated as the code changes. If you look at the code below, you know what that stuff makes. In my opinion, these comments is useless. But if you think differently, I invite you to discuss it.

Reply:
Normally, I would abstain from comments, here however I decided to act on a technique I've observed that's specific to unit tests. It separates the test into three clear sections of given-when-then, making the procedure easier to read by introducing a transparent and repeatable pattern of structure.
I understand though that it might be a niche approach, and we can forgo it within this application if need be.

https://en.wikipedia.org/wiki/Given-When-Then